Modules/Loadable/Transforms/MRMLDM/vtkMRMLTransformsDisplayableManager3D.cxx

```diff
@@ -47,6 +53,9 @@
 #include <vtkRenderWindowInteractor.h>
 #include <vtkRenderer.h>
 #include <vtkSmartPointer.h>
+#include <vtkTransform.h>
+
+const double DEFAULT_SCALE[3] = {100.0, 100.0, 100.0};
```

Comment:
Good catch ! It's a leftover. I need a better clean up of this displayable manager.